### ISSUE TITLE
SyntaxError: invalid syntax in d2khelper.py

### ISSUE THREAD

**Issue by Motorrat** [paypal/autosklearn-zeroconf#9]
freshly installed CentOS7 

```
 python3 ./bin/zeroconf.py -d ./data/Adult.h5

Traceback (most recent call last):
  File "./bin/zeroconf.py", line 22, in <module>
    import d2khelper as d2k
  File "/home/username/autosklearn-zeroconf/bin/d2khelper.py", line 48
    return {**parameter, **param}
             ^
SyntaxError: invalid syntax
```
@d2k do you you think you could look into this?

**Comment by Motorrat** [comment]
actual installation script
```
 1  sudo yum -y update
    2  sudo yum install epel-release python34 python34-devel python34-setuptools
    4  sudo yum -y groupinstall 'Development Tools'yum -y groupinstall 'Development Tools'
    8  wget https:/downloads.sourceforge.net/project/swig/swig/swig-3.0.12/swig-3.0.12.tar.gz -O swig-3.0.12.tar.gz
   11  tar xf swig-3.0.12.tar.gz
   12  cd swig-3.0.12
   13  ./configure --without-pcre
   15  make; sudo make install
   16  sudo pip3 install virtualenv
   17  cd ..
   18  sudo easy_install-3.4 pip
   19  sudo pip3 install virtualenv
   20  virtualenv zeroconf -p /usr/bin/python3.4
   21  source zeroconf/bin/activate
   23  pip install --upgrade cython
   24  curl https://raw.githubusercontent.com/paypal/autosklearn-zeroconf/master/requirements.txt | xargs -n 1 -L 1 pip install
   26  sudo yum  install git gcc build-essential swig
   27  git clone https://github.com/paypal/autosklearn-zeroconf.git
   29  cd autosklearn-zeroconf/
   31  python3 ./bin/zeroconf.py -d ./data/Adult.h5

```

**Comment by d2k** [comment]
The issue is related to python 3.4.
The {**parameter, **param} syntax to merge two dicts is requiring 3.4 
Will change it to 3.4 syntax

**Comment by Motorrat** [comment]
@d2k please consider forking this repository again before making your changes to keep all changes I have done to README etc since your pull request.

**Comment by d2k** [comment]
fixed with 3.4 syntax. please accepot the pull request
